spec/datetimeformat.html

```diff
@@ -458,6 +500,24 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-Intl.DateTimeFormat.prototype.formatToParts">
+      <h1>Intl.DateTimeFormat.prototype.formatToParts ([ date ])</h1>
+
+      <p>
+        When the *Intl.DateTimeFormat.prototype.formatToParts* is called with an optional argument _date_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _dtf_ be *this* value.
+        1. Assert: Type(_dtf_) is Object and _dtf_ has an [[initializedDateTimeFormat]] internal slot whose value is *true*.
+        1. If _date_ is not provided or is *undefined*, then
+          1. Let _x_ be *%Date_now%*().
+        1. Else,
+          1. Let _x_ be ? ToNumber(_date_).
+        1. Return ? FormatDateTimeToParts(_dtf_, _x_).
```

Comment:
So, @stasm removed it in his editorial patch for my PR - https://github.com/stasm/ecma402/commit/3b35e0b63dd99bf8df350a96f80b349aef8d1f5f

I'll bring it back.